Prefetching-Library/android_prefetching_lib/src/androidTest/java/nl/vu/cs/s2group/nappa/room/dao/ActivityVisitTimeDaoTest.java

```diff
@@ -59,31 +55,4 @@ public void tearDown() throws Exception {
         db.close();
     }
 
-    @Test
-    public void testGetActivityVisitTimeByActivity() {
-        List<ActivityVisitTime> result = db.activityVisitTimeDao().getActivityVisitTime(1L);
-        assertThat(result.size(), equalTo(activityList.size() * 10));
-        assertThat(result.get(0).timestamp, equalTo(timeList.get(0).timestamp));
-        assertThat(result.get(0).duration, equalTo(timeList.get(0).duration));
-    }
-
-    @Test
-    public void testGetActivityVisitTimeByActivityAndSession() {
-        List<ActivityVisitTime> result = db.activityVisitTimeDao().getActivityVisitTime(1L, 1L);
-        assertThat(result.size(), equalTo(10));
-        assertThat(result.get(0).timestamp, equalTo(timeList.get(0).timestamp));
-        assertThat(result.get(0).duration, equalTo(timeList.get(0).duration));
-    }
-
-    @Test
-    public void testGetActivityVisitTimeByActivityAndLastSession() {
-        List<ActivityVisitTime> result = db.activityVisitTimeDao().getActivityVisitTime(1L, 3);
-        assertThat(result.size(), equalTo(30));
-
-        result = db.activityVisitTimeDao().getActivityVisitTime(1L, 2);
-        assertThat(result.size(), equalTo(20));
-
-        result = db.activityVisitTimeDao().getActivityVisitTime(1L, 1);
-        assertThat(result.size(), equalTo(10));
-    }
```

Comment:
These tests test queries that are no longer needed and thus are removed (with the queries).

As it is, the tests are not very useful (due to the manual creation of data). The tests need static data (e.g., a DB file from running the library in an app) to precisely verify the correctness.